### PR TITLE
[WIP] Support skim as an alternative to fzf

### PR DESCRIPTION
**Pull request by jedahan**
First draft at #115

### CODE REVIEW

File: src/ui.sh [hunk 1]
```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+ui::fzf_cmd() {
+  [ "$NAVI_ENV" == "test" ] && echo fzf_mock && return
+  command_exists sk && echo sk && return
+  echo "fzf -i"
```
Comment (jedahan) [inline src/ui.sh]: untested, kinda hacky

Comment (denisidoro) [reply src/ui.sh]: I'm not sure I got it... You're changing the behavior for `fzf` here? Doesn't it work as-is?

**Comment by denisidoro** [comment]
Related: https://github.com/denisidoro/navi/issues/115

File: src/ui.sh (hunk 1 above)
Comment (jedahan) [reply src/ui.sh]: Actually its the opposite - i'm removing "-i" from skim, but keeping it in `fzf`.

**Comment by denisidoro** [comment]
Closing without merging because I don't plan to support skim in the bash version of navi. Maybe I will in the Rust version I'm planning to write.

Feel free to reopen the PR, though!

**Pull request closed without merging**